app/assets/stylesheets/administrate/base/_forms.scss

```diff
@@ -1,78 +1,99 @@
 fieldset {
-  background: lighten($base-border-color, 10);
-  border: $base-border;
-  margin: 0 0 ($base-spacing / 2) 0;
-  padding: $base-spacing;
+  background-color: transparent;
+  border: 0;
+  margin: 0;
+  padding: 0;
 }
 
-input,
-label,
-select {
-  display: block;
-  font-family: $form-font-family;
-  font-size: $form-font-size;
+legend {
+  font-weight: 600;
+  margin-bottom: $small-spacing / 2;
+  padding: 0;
 }
 
 label {
-  font-weight: bold;
-  margin-bottom: $base-spacing / 4;
-
-  &.required:after {
-    content: "*";
-  }
+  display: block;
+  font-weight: 600;
+  margin-bottom: $small-spacing / 2;
+}
 
-  abbr {
-    display: none;
-  }
+input,
+select {
+  display: block;
+  font-family: $base-font-family;
+  font-size: $base-font-size;
 }
 
+[type="color"],
+[type="date"],
+[type="datetime"],
+[type="datetime-local"],
+[type="email"],
+[type="month"],
+[type="number"],
+[type="password"],
+[type="search"],
+[type="tel"],
+[type="text"],
+[type="time"],
+[type="url"],
+[type="week"],
+input:not([type]),
 textarea,
-#{$all-text-inputs},
-select {
-  background-color: white;
-  border-radius: $form-border-radius;
-  border: 1px solid $form-border-color;
-  box-shadow: $form-box-shadow;
+select[multiple] {
+  background-color: $white;
+  border: $base-border;
+  border-radius: $base-border-radius;
+  box-shadow: none;
   box-sizing: border-box;
-  font-family: $form-font-family;
-  font-size: $form-font-size;
+  font-family: $base-font-family;
+  font-size: $base-font-size;
   margin-bottom: $small-spacing;
-  padding: $form-input-padding;
-  transition: border-color;
+  padding: $base-spacing / 3;
+  transition: border-color $base-duration $base-timing;
   width: 100%;
 
   &:hover {
-    border-color: $form-border-color-hover;
+    border-color: shade($base-border-color, 20%);
   }
 
   &:focus {
-    border-color: $form-border-color-focus;
-    box-shadow: $form-box-shadow-focus;
+    border-color: $action-color;
+    box-shadow: none;
     outline: none;
   }
+
+  &:disabled {
+    background-color: shade($base-background-color, 5%);
+    cursor: not-allowed;
+
+    &:hover {
+      border: $base-border;
+    }
+  }
 }
 
 textarea {
   resize: vertical;
 }
 
-input[type="search"] {
+[type="search"] {
   appearance: none;
 }
 
-input[type="checkbox"],
-input[type="radio"] {
+[type="checkbox"],
+[type="radio"] {
   display: inline;
-  margin-right: $base-spacing / 4;
+  margin-right: $small-spacing / 2;
```

Comment:
I think Hound fired off on all these operator spaces because it’s using [SCSS-Lint v42](https://github.com/thoughtbot/hound-scss/blob/master/Gemfile#L7) and I have v43 configured via the setup here locally in this repo…?